pocket/assetlist.json

```
@@ -0,0 +1,42 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "pocket",
+  "assets": [
+    {
+      "description": "The native token of Pocket Network",
+      "denom_units": [
+        {
+          "denom": "upokt",
+          "exponent": 0
+        },
+        {
+          "denom": "pokt",
+          "exponent": 6
+        }
+      ],
+      "base": "upokt",
+      "name": "POKT",
+      "display": "pokt",
+      "symbol": "POKT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.svg"
+      },
+      "keywords": ["rpc", "api"],
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.svg",
+          "theme": {
+            "circle": true
```

```suggestion
            "circle": true,
            "primary_color_hex": "#0C3B9D"
```

Color of the blue logo.